server/repository/src/db_diesel/patient.rs

```diff
@@ -517,4 +538,57 @@ mod tests {
             .unwrap();
         assert!(!result.is_empty());
     }
+
+    #[actix_rt::test]
+    async fn test_name_date_of_death() {
```

Comment:
I only focused on the "is alive" queries in this test. Let me know if you think I should test the "normal" date query cases as well.

Reply:
Looks good to me, thanks